### PR TITLE
migrating vtctld-web python testcase to go

### PR DESCRIPTION
**Pull request by princeparmar**
1. created selenium setup method.
2. used vtcombo for the cluster setup.

**list of testcase**

> TestRealtimeStats checks the status by changing dropdown values.
> TestShardView validates tablet type and uids.
> TestKsView validates the shard names for keyspace.
> TestCreateKs validates the keyspace creation using ui.
> TestDashboard validate the keyspaces and shard in dashboard.
> TestDashboardValidate validates the validate command from the ui.

### CODE REVIEW

**Comment by deepthi** [comment]
@morgo what would be the right way to include `chromedriver`? 
```
2020/02/18 11:09:03 local.web_test: running tests for vitess.io/vitess/go/test/endtoend/vtctldweb 
fork/exec /home/runner/work/vitess/vitess/dist/chromedriver/chromedriver: no such file or directory
FAIL	vitess.io/vitess/go/test/endtoend/vtctldweb	0.009s
```

**Comment by morgo** [comment]
> @morgo what would be the right way to include `chromedriver`?
> 
> ```
> 2020/02/18 11:09:03 local.web_test: running tests for vitess.io/vitess/go/test/endtoend/vtctldweb 
> fork/exec /home/runner/work/vitess/vitess/dist/chromedriver/chromedriver: no such file or directory
> FAIL	vitess.io/vitess/go/test/endtoend/vtctldweb	0.009s
> ```

The PR now just uncomments the feature-gating on installing chromedriver (`BUILD_PYTHON`). I checked out how long it takes, and it looks to be really quick.. so I think this is fine.

The other way to do it would be to create a new feature gate (`BUILD_CHROMEDRIVER`). But I don't think we need to.

File: go/test/endtoend/vtctldweb/vtctld_web_main_test.go [hunk 1]
```diff
@@ -0,0 +1,509 @@
...
+	"github.com/stretchr/testify/require"
+	"github.com/tebeka/selenium"
+	"github.com/tebeka/selenium/chrome"
+	vttestpb "vitess.io/vitess/go/vt/proto/vttest"
+	"vitess.io/vitess/go/vt/vttest"
+)
+
+var (
+	localCluster    *vttest.LocalCluster
+	cell            = "zone1"
```
Comment (princeparmar) [inline go/test/endtoend/vtctldweb/vtctld_web_main_test.go]: need to remove cell, it is not used.

**Comment by morgo** [comment]
> Looks ok overall. @morgo any concerns with running the test inside docker?

Not from me. I am also good with this being a separate github action as it now is. We will likely have different dependencies as we test the GUI in the future.

File: .github/workflows/cluster_vtctl_web.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        go mod download
+
+    - name: Run make tools
+      run: |
+        make tools
+
+    - name: Run vtctl web
+      run: |
+        # Running web test inside docker
+        eatmydata -- go run test.go -docker=true -print-log -shard 25
```
Comment (morgo) [inline .github/workflows/cluster_vtctl_web.yml]: Nit: The eatmydata won't propagate to the Docker container. I suggest we remove it from here (and the apt-get install line above) so it doesn't confuse someone down the road.

**Pull request merged**